ext/drreg/drreg.c

```diff
@@ -1838,6 +2762,16 @@ tls_data_init(per_thread_t *pt)
         drvector_init(&pt->reg[GPR_IDX(reg)].live, 20, false /*!synch*/, NULL);
         pt->reg[GPR_IDX(reg)].native = true;
     }
+#ifdef X86
+    for (reg = DR_REG_APPLICABLE_START_SIMD; reg <= DR_REG_APPLICABLE_STOP_SIMD; reg++) {
+        drvector_init(&pt->simd_reg[SIMD_IDX(reg)].live, DR_NUM_SIMD_VECTOR_REGS,
+                      false /*!synch*/, NULL);
+        pt->simd_reg[SIMD_IDX(reg)].native = true;
+    }
+    /* Block should be aligned on a 64-byte boundary */
+    pt->simd_spill_start = dr_global_alloc((REG_SIMD_SIZE * MAX_SIMD_SPILLS) + 63);
```

Comment:
I think we can use thread_alloc instead of global alloc since the indirect block is thread local.

Reply:
Also is there an ALIGN macro visible to drreg which we can use here?

Reply:
Good idea. Done!